src/resources/MachineLearning/Models/Models.ts

```diff
@@ -7,8 +7,11 @@ import {MLModel} from './ModelsInterfaces';
 export default class Models extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/models`;
 
-    list() {
-        return this.api.get<MLModel[]>(Models.baseUrl);
+    list(engines?: string[]) {
+        const requestUrl = Boolean(engines)

```

Comment:
`[]` is truthy, meaning that the `engines` query parameter will be added even when no engines are specified. We might want to validate the array length as well.

Reply:
ah oups ill check for length instead thanks